tests/integration/session_integration/config.yml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
An empty `config.yml` is unnecessary.

Reply:
Fixed in 36e402566cbeb3d99e3bb7f268abd41e6a87b8f9